cadc-postgresql-dev/README.md

```diff
@@ -8,33 +8,30 @@ Note: the PostgreSQL 10.x build includes the pgsphere extension.
 TODO: add pgsphere extension to the PostgreSQL 12.x build once there is an rpm available from CADC.
 
 ## databases
-On startup, the following databases are created:
+On startup, the following user accounts are created (name : password):
 ```
-cadctest : intended for database library integration tests
-content  : intended for web service back end
-```
-
-## users
-The following user accounts are created (name : password):
-```
-cadmin : pw-cadmin
-tapadm : pw-tapadm
+cadmin  : pw-cadmin
+tapadm  : pw-tapadm
 tapuser : pw-tapuser
 ```
 These users are available in all databases.
 
-## schemas
-The following schemas are created (name : acccount with full permissions):
+## databases and schemas
+The following schemas are created in all databases (name : acccount with full authorization):
 ```
-tap_schema : tapadm
 tap_upload : tapuser
-uws : tapadm
+tap_schema : tapadm
+uws        : tapadm
 ```
-These schemas are available in all databases. Addtional schemas can be configured by including the file
-/config/init-content-schemas.sh with
+
+Databases to be created, and additional schemas for content, are configured by including the 
+file /config/init-content-schemas.sh:
 ```
+CATALOGS="db1 db2 ..."
```

Comment:
CATALOG instead of DATABASE? Is catalog a postgresql term for database? It's made clear below that catalog == database, so I won't have a problem with catalog, by why not just database?